api/client/samples/crop_w_series/top_5_crop_w_series.ipynb

```diff
@@ -0,0 +1,1415 @@
+{
```

Comment:
<p>Could you make it clear that the <code>weighted_source</code>  variable would be a str, or the name of a source?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/312/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>